cmake/MacroHelper.cmake

```
@@ -207,13 +207,17 @@ endmacro()
 # Run nocmodl to convert NMODL to C
 # =============================================================================
 macro(nocmodl_mod_to_cpp modfile_basename)
+  set(NOCMODL_SED_EXPR "s/_reg()/_reg_()/")
+  if(NOT MSVC)
```

a nitpick: 
```suggestion
  if(NRN_WINDOWS_BUILD)
```
Maybe this is better?

---

MINGW runs everything in `bash` and is ignorant of this

---

Ok. I was thinking of other compilers on windows native build (if they exist! 😅 )